### PR TITLE
Text/Image Search Revamp

### PR DESCRIPTION
**Pull request by cocovsack**
Completely redoes the user search experience. Adds text search that takes users directly to nutrition information. Also separates the image/text search functionality from the homepage.

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -30,22 +30,11 @@
...
-          <i onclick="closeImageForm()" class="icon-remove-sign icon-2x"></i>
-          <img class="image-preview" id="image-preview" >
-          <br>
-          <button type="submit" id="upload" class="upload">Upload</button>
-        </form>
+        <h3 id="about-text-title">About Forked</h3>
+        <h4 id="about-text" class="about-text">Curious what ingredients are in your dish? Want to find a similar recipe for later? Worried about potential food allergens or restrictions? Forked provides you with instantanous information on your dish. Just upload your photo below and let us do the rest!</h4>
+        <button class="button-container" onclick="window.location.href='/imageform.html'"> <i class="icon-picture search-icon"></i></button>
+        <button class="button-container" onclick="window.location.href='/textform.html'"> <i class="icon-font search-icon"></i></button>
+        <!-- <button id="popup-button" class="open-button" ">Analyze Your Dish</button> -->
```
Comment (liguanghua) [inline src/main/webapp/index.html:37]: Delete the unused line.

File: src/main/webapp/script.js [hunk 2]
```diff
@@ -225,27 +225,38 @@ function startSlideshow() {
...
 function previewImage(input) {
+  var preview = document.getElementById("image-preview");
+  var label =  document.getElementById("label-title");
+  var container = document.getElementById("input");
+  var reader = new FileReader();
+  var upload = document.getElementById("upload");
   if(input.files && input.files[0]) {
-    preview = document.getElementById("image-preview")
-    var reader = new FileReader();
+    container.style.padding = "20px 20px 30% 20px";
```
Comment (linxiz) [inline src/main/webapp/script.js:254]: could this live in css file?

File: src/main/webapp/style.css [hunk 3]
```diff
@@ -219,6 +219,28 @@ body {
...
 }
 
+.search-icon {
+  font-size: 130px;
+  color: darkseagreen;
+  opacity: 0.7;
+  padding-left: 60px;
+  padding-right: 60px;
+  padding-top: 50px;
+  text-shadow: 0 0 2px black;
```
Comment (linxiz) [inline src/main/webapp/style.css:229]: nit: order alphabetically and ditto below

File: src/main/webapp/style.css [hunk 4]
```diff
@@ -517,47 +537,27 @@ h2 {
...
-  text-decoration: underline;
-  top: 7%;
-  width: 30%;
-}
-
-.file-container:hover {
+/* .file-container:hover {
   opacity: 1;
-}
+} */
```
Comment (linxiz) [inline src/main/webapp/style.css:542]: remove if not used

File: src/main/webapp/script.js (hunk 2 above)
Comment (cocovsack) [reply src/main/webapp/script.js:254]: No. It does live in CSS but needs to be modified in certain case that is why this change happens in javascript.

**Pull request merged**